### PR TITLE
add red-button class to navbar buttons

### PR DESCRIPTION
**Pull request by InouRidder**
I added a red button class in the buttons.scss, 
added the class to the navbar.html.erb
Please review!

### CODE REVIEW

**Comment by MadihaLatafi** [comment]
Amazing buttons !

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -4,15 +4,21 @@
     <%= image_tag "logo.png", height: 50 %>
   <% end %>
 
+<style>
+  .red-button {
+    background-color: red;
```
Comment (MadihaLatafi) [inline app/views/shared/_navbar.html.erb:9]: let's make it green !

File: app/assets/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.red-button {
+  background-color: red;
```
Comment (InouRidder) [inline app/assets/stylesheets/components/_buttons.scss:2]: why didnt you add some opacity?

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -4,15 +4,21 @@
...
 
     <% if user_signed_in? %>
 
       <!-- Links when logged in -->
-      <%= link_to "Host", "#", class: "navbar-wagon-item navbar-wagon-link" %>
-      <%= link_to "Trips", "#", class: "navbar-wagon-item navbar-wagon-link" %>
-      <%= link_to "Messages", "#", class: "navbar-wagon-item navbar-wagon-link" %>
+      <%= link_to "Host", "#", class: "navbar-wagon-item navbar-wagon-link red-button" %>
+      <%= link_to "Trips", "#", class: "navbar-wagon-item navbar-wagon-link red-button" %>
+      <%= link_to "Messages", "#", class: "navbar-wagon-item navbar-wagon-link red-button" %>
```
Comment (InouRidder) [inline app/views/shared/_navbar.html.erb:21]: maybe one button less with this class?

File: app/assets/stylesheets/components/_buttons.scss (hunk 2 above)
Comment (MadihaLatafi) [reply app/assets/stylesheets/components/_buttons.scss:2]: Agreed !

**Pull request merged**